acceptance/README.md

```diff
@@ -1,257 +1,450 @@
-The rake task that we'll use is `ci:test:git`.
-Run
-```
-bundle exec rake -D ci:test:git
-```
-to read the full description of this task.
-From the description, we can see that we'll need to set a few environment variables:
-  + CONFIG should be set to point to the CentOS beaker config file we created above.
-  + SHA should be the SHA of the commit we want to test.
-  + SERVER should be the IP address of the host (i.e. your machine) in the vagrant private network created for the test box.
-    This is derived from the test box's ip by replacing the last octet with 1.
-    For our example above, the host IP is 192.168.80.1
-  + FORK should be the path to a 'puppetlabs-puppet.git' directory that points to the repo.
-    In our case, this is the path to the symlink we created before, which is inside your puppet repo checkout, so FORK should just be the name of your checkout.
-    We'll assume that the name is `puppet`.
+### Rerunning Failed Tests
+The rake tasks we use here take advantage of a newer feature in beaker that gives us quite a bit of flexibility. We take advantage of beaker subcommands. Subcommands are individual beaker invocations that are used to run the different stages of running tests: provisioning, pre-suite setup, tests, etc. We do this by writing state to the file `.beaker/subcommand_options.yaml`. With each new invocation of a subcommand, beaker will check for this file and load the contents if the file exists. The important thing about this feature is that you can rerun tests without going through the entire provisioning process every time.
 
-Putting it all together, we construct the following command-line invocation to run the tests:
-```sh
-CONFIG=config/nodes/centos6-local.yaml SHA=#{test-commit-sha} SERVER='192.168.80.1' FORK='puppet' bundle exec rake --trace ci:test:git
+To ensure your hosts aren't cleaned up after a run, set `OPTIONS='--preserve-hosts=always'`. With this set, we can rerun a failed test using the infrastructure beaker has already provisioned.
 ```
-Go ahead and run that sucker!
-
-Testing will take some time.
-After the testing finishes, you'll either see this line
+bundle exec rake ci:test:aio OPTIONS='--preserve-hosts=always' SHA=6.0.5
+```
+If this run fails because a small handful of tests fail, I can rerun only those tests that failed. For example, assume that `tests/resource/package/yum.rb` and `tests/node/check_woy_cache_works.rb` both had failing tests. I can run
 ```
-systest completed successfully, thanks.
+bundle exec beaker exec tests/resource/package/yum.rb,tests/node/check_woy_cache_works.rb
 ```
-near the end of the output, indicating that all tests completed successfully, or you'll see the end of a stack trace, indicating failed tests further up.
+
+This will work regardless of which hypervisor or testing method you are using.
+
+-------------
+
+## Writing Tests
+* TODO I'm definitely going to need help with this section. I'm not very good at writing tests. Jacob?
```

Comment:
@jhelwig or @joshcooper any resources or deets that I should put here?

Reply:
Since nothing's been added here, let's just update the TODO note to be more generalized without tagging specific people in the doc. :) We can add to this section later after your initial changes have been merged.